### ISSUE TITLE
wrong label data of back_azimuth_deg

### ISSUE THREAD

**Issue by Seismic-wave-Han** [smousavi05/STEAD#8]
Hello. 
I'm Sangwoo Han in Seismology, Geophysics and Tectonophysics Lab in Seoul National Univ. in South Korea.
First of all, thank you for your effort to make a very cool global benchmark dataset.

I found some problems in the label data of 'back_azimuth_deg'.
There are some 180-degree differences between 'back_azimuth_deg' in STEAD and back-azimuth obtained using locations of source and receiver.
I download STEAD data from mega cloud link.
I attached an image of back-azimuth differences.
![figure](https://user-images.githubusercontent.com/48647491/143680568-c1fdd1cb-15d9-4be3-b5f8-778b1602b2d3.png)

Also, I gave you one example of this problem.
- Trace name: 'AKGG.AV_20060814223729_EV'

I used function gps2dist_azimuth() from obspy.geodetics.base to get back-azimuth.
I also want to know the formula that you used to calculate 'back_azimuth_deg'
Please check this problem and could you reply to this?

**Comment by smousavi05** [comment]
Hi @Seismic-wave-Han , you are right. The reported value for 'AKGG.AV_20060814223729_EV' is azimuth not the back azimuth.I used the same function from Obspy. And this is how it was calculated during the compiling of the dataset:

                distanceKm, azimuth, back_azimuth = obspy.geodetics.base.gps2dist_azimuth(
                                                                            float(event_lat), 
                                                                            float(event_lon),
                                                                            float(station_lat), 
                                                                            float(station_lon), 
                                                                            a=6378137.0, 
                                                                            f=0.0033528106647474805)
                distanceKm = (distanceKm/1000);
                distance = round((distanceKm / 111), 4);
                back_azimuth = round(back_azimuth, 4)

However, I haven't figured out what went wrong yet. I checked the station lat and long and they are correct. So no flipping of the event-station coordinates. Could you tell me how many of these you found in the dataset?

**Comment by Seismic-wave-Han** [comment]
Hi @smousavi05. I prepared some figures related to the error of back-azimuth (also, distance)
![baz_error](https://user-images.githubusercontent.com/48647491/143849561-d033beaf-7145-4ec3-a4e3-8eb18640db4d.png)
![baz_error_hist](https://user-images.githubusercontent.com/48647491/143849598-fd7346c3-4b73-445f-b724-41ee226c4a14.png)
These figures are the back-azimuth error along with whole samples and their histogram.
Samples with errors in the 170-190 degree range are 8% of the total.

I also found the distance discrepancy with obspy.
![dist_error](https://user-images.githubusercontent.com/48647491/143850578-ae17afd1-3a87-4482-a4fb-2bb9e2b5bafd.png)
The error of distance is nearly zero, but it shows quite a large error for some samples.
I hope this information will help you find systematic errors in STEAD.
